specification/dnc/resource-manager/readme.md

```diff
@@ -97,22 +111,36 @@ output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-DelegatedNetwork
 
 ``` yaml $(java) && $(multiapi)
 batch:
+  - tag: package-2021-03-15
   - tag: package-2020-08-08-preview
 ```
 
-### Tag: package-2020-08-08-preview and java
+### Tag: package-2021-03-15 and java
 
-These settings apply only when `--tag=package-2020-08-08-preview --java` is specified on the command line.
+These settings apply only when `--tag=package-2021-03-15 --java` is specified on the command line.
 Please also specify `--azure-libraries-for-java=<path to the root directory of your azure-sdk-for-java clone>`.
 
-``` yaml $(tag) == 'package-2020-08-08-preview' && $(java) && $(multiapi)
+``` yaml $(tag) == 'package-2021-03-15' && $(java) && $(multiapi)
 java:
-  namespace: com.microsoft.azure.management.DelegatedNetwork.2020-08-08-preview
-  output-folder: $(azure-libraries-for-java-folder)/sdk/DelegatedNetwork/mgmt-2020-08-08-preview
+  namespace: com.microsoft.azure.management.DelegatedNetwork.2021-03-15
+  output-folder: $(azure-libraries-for-java-folder)/sdk/DelegatedNetwork/stable/mgmt-2021-03-15
 regenerate-manager: true
 generate-interface: true
 ```
 
+### Tag: package-2020-08-08-preview and java
+
+These settings apply only when `--tag=package-2020-08-08-preview --java` is specified on the command line.
+Please also specify `--azure-libraries-for-java=<path to the root directory of your azure-sdk-for-java clone>`.
+
+```yaml $(tag) == 'package-2020-08-08-preview' && $(java) && $(multiapi)
+java:
+  namespace: com.microsoft.azure.management.DelegatedNetwork.v2020_08_08_preview
+  output-folder: $(azure-libraries-for-java-folder)/sdk/DelegatedNetwork/preview/mgmt-v2020_08_08_preview
+  regenerate-manager: true
+  generate-interface: true
+```
```

Comment:
Could we extract the java configurations to its own file?

Reply:
done